package.json

```diff
@@ -261,6 +261,7 @@
     "react-is": "18.3.1",
     "react-keyed-flatten-children": "1.2.0",
     "react-modal": "3.15.1",
+    "react-native": "0.74.1",
```

Comment:
These new dependencies should match exactly to the versions in the app package.json.